### DISCUSSION TITLE
Issue filter glyph misaligned

### DISCUSSION THREAD

**Discussion by KalleOlaviNiemitalo** [community/community#6408] [Mobile] [answered]
I defined custom filters named "🤖 Bot" and "⛏ Minecraft". Now when I view the list of filters in the GitHub 1.33.0 app on iOS 15.0.2, the 🤖 glyph (U+1F916) is aligned with the glyphs of the default filters, but the "⛏" glyph (U+26CF) is instead aligned with the names of the other filters:

![image](https://user-images.githubusercontent.com/46201428/137577485-76fd492b-e0bb-49da-93f7-74cf19da2971.png)

On the <https://github.com/notifications> page though, all glyphs are nicely lined up.

**Comment by KalleOlaviNiemitalo** [comment]
Seems to work better with "⛏️ Minecraft" (U+26CF U+FE0F):

![image](https://user-images.githubusercontent.com/46201428/137577878-ba22393e-4cfc-4694-8881-8a4d85b11a69.png)

Still, it's odd that the variation selector would make such a difference, when the glyph itself looks the same either way.

**Comment by amrehman** [comment, marked as answer]
Hi there,
This definitely sounds like a bug. Sorry about this, I am going to file an issue for our dev team straight away.

Thanks a lot for reaching out!
